cmdapp/src/python.rs

```diff
@@ -23,6 +25,93 @@ fn convert_image_to_svg_py(
     let input_path = PathBuf::from(image_path);
     let output_path = PathBuf::from(out_path);
 
+    let config = construct_config(
+        colormode,
+        hierarchical,
+        mode,
+        filter_speckle,
+        color_precision,
+        layer_difference,
+        corner_threshold,
+        length_threshold,
+        max_iterations,
+        splice_threshold,
+        path_precision,
+    );
+
+    convert_image_to_svg(&input_path, &output_path, config).unwrap();
+    Ok(())
+}
+
+#[pyfunction]
+fn convert_py(
+    img_bytes: Vec<u8>,
+    img_format: Option<&str>,      // Format of the image. If not provided, the image format will be guessed based on its contents. 
```

Comment:
what are the possible image format? e.g. `rgba8888` ?

can you give some examples?

Reply:
These are image file formats supported by the `image` crate, listed [here](https://docs.rs/image/latest/image/enum.ImageFormat.html)